pkgs/tools/admin/meraki-cli/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, argcomplete
+, jinja2
+, meraki
+, rich
+, fetchPypi
+, buildPythonApplication
+, pytestCheckHook
+, requests-mock
+}:
+
+buildPythonApplication rec {
+  pname = "meraki-cli";
+  version = "1.5.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    pname = "meraki_cli";
+    inherit version;
+    hash = "sha256-YOyeovqRqt6ZMXgLnIxRvPkcW259K8NIBGdb3PwjkMg=";
+  };
+
+  # requires files not in PyPI tarball
+  disabledTests = [
+    "TestDocVersions"
+    "TestHelps"
+    "TestUpgrade"
+  ];
```

```suggestion
  disabledTests = [
    # requires files not in PyPI tarball
    "TestDocVersions"
    "TestHelps"
    # requires running "pip install"
    "TestUpgrade"
  ];
```

I think this comment needs updating now that another test class has been disabled

I think this should be last change and I'll merge tonight assuming OfBorg looks green (and I'll probably test aarch64-darwin myself if it's the only build not finished by then)

---

Done